src/mmw/requirements/base.txt

```diff
@@ -28,3 +28,4 @@ numpy==1.13.0
 hs_restclient==1.2.10
 six==1.11.0
 fiona==1.7.11
+timeout-decorator==0.4.0
```

Comment:
We talked about the risks of switching from Gunicorn sync workers vs. async workers. It looks like this library is an example of one that would be difficult to use with async workers due to its use of signals.

See: https://github.com/pnpnpn/timeout-decorator#multithreading

It looks like we're in the clear, but definitely worth exercising the timeout process multiple times in one session, with the service launched via Upstart vs. the `debugserver` script.